crates/pera-core/src/dwallet_mpc/network_dkg.rs

```diff
@@ -0,0 +1,155 @@
+use crate::authority::authority_per_epoch_store::AuthorityPerEpochStore;
+use crate::dwallet_mpc::dkg::DKGFirstParty;
+use crate::dwallet_mpc::mpc_instance::{DWalletMPCInstance, MPCSessionStatus};
+use crate::dwallet_mpc::mpc_party::{advance, MPCParty};
+use commitment::CommitmentSizedNumber;
+use dwallet_mpc_types::ClassGroupsPublicKeyAndProof;
+use group::PartyID;
+use homomorphic_encryption::AdditivelyHomomorphicDecryptionKeyShare;
+use jsonrpsee::core::Serialize;
+use mpc::WeightedThresholdAccessStructure;
+use pera_types::base_types::{EpochId, ObjectID};
+use pera_types::error::{PeraError, PeraResult};
+use pera_types::messages_dwallet_mpc::{MPCRound, SessionInfo};
+use serde::Deserialize;
+use std::collections::HashMap;
+use std::sync::Arc;
+
+const NONE_OBJ_ID: ObjectID = ObjectID::from_single_byte(0);
+const SECP256K1_DKG_SESSION_ID: ObjectID = ObjectID::from_single_byte(0);
```

Comment:
Ye this is problematic, let's go with something like
```suggestion
const SECP256K1_DKG_SESSION_ID: ObjectID = ObjectID::bytes("chain-dkg".to_bytes());
```